cogs5e/models/homebrew/bestiary.py

```diff
@@ -362,9 +365,22 @@ def parse_critterdb_traits(data, key):
 
         if overrides:
             for override in overrides:
-                attacks.append({'name': override.group(1) or name,
-                                'attackBonus': override.group(2) or None, 'damage': override.group(3) or None,
-                                'details': desc})
+                try:
+                    attack_yaml = yaml.safe_load(override.group(1))
+                    if not isinstance(attack_yaml, list):
+                        attack_yaml = [attack_yaml]
+                    if not all(isinstance(x, dict) for x in attack_yaml):
+                        raise ValueError('Invalid YAML')
+                    conflicts = [a for a in attacks if a['name'].lower() in [new['name'].lower() for new in attack_yaml]]
+                    for a in conflicts:
+                        attacks.remove(a)
```

Comment:
So I believe standard for normal attack lists for PC's would be to append a number to the 'duplicate' attacks, so like Longsword, Longsword2, Longsword3.

You could also account for them missing the name key by grabbing the name of the action, which is I believe the default for basic attack overrides atm

Reply:
On the first point:

The current system actually has no conflict detection to stop you giving a CritterDB creature multiple attacks with the same name. I think it's a very different context from character sheet imports (a basic and widely used feature) but this does seem like an improvement on how it currently works so I've added it in.

On the second point:

Makes sense. I've added this in.